### ISSUE TITLE
Front End: Add more sizes

### ISSUE THREAD

**Issue by shawnadelic** [hackforla/spare#135]
It seems like we're probably missing some common sizes (for example, Mens' shoes only go to 11).

No need to go too extreme, but should be a little more accommodating for accessibility's sake.

**Comment by Allicolyer** [comment]
what about adding an "other" field or a "custom" field so that people could add a size they do not see?

**Comment by Allicolyer** [comment]
One more idea here, what about adding kid sizes?
